src/main/resources/resources/logisim/strings/gui/gui_pl.properties

```diff
@@ -332,6 +332,11 @@ editPasteItem = Wklej
 editRedoItem = Ponów %s
 editSelectAllItem = Zaznacz wszystko
 editUndoItem = Cofnij %s
+editUndoHistoryMenu=Historia Cofnij
+editRedoHistoryMenu=Historia Ponów
+editClearHistoryAction=Wyczyść historię Cofnij/Ponów
+clearHistoryWarningTitle=Wyczyść historię
+clearHistoryWarningMessage=Czy na pewno chcesz trwale wyczyścić historię Cofnij/Ponów?\nTej operacji nie można cofnąć.
```

Comment:
Use lower-case for all occurrences of "Cofnij" and "Ponów":

"Cofnij" -> "cofnij"
"Ponów" -> "ponów"